cluster/gce/util.sh

```diff
@@ -868,7 +890,13 @@ function construct-linux-kubelet-flags {
   else # For nodes
     flags+=" ${NODE_KUBELET_TEST_ARGS:-}"
     flags+=" --bootstrap-kubeconfig=/var/lib/kubelet/bootstrap-kubeconfig"
-    flags+=" --kubeconfig=/var/lib/kubelet/kubeconfig"
+    
+    if [[ "${ARKTOS_SCALEOUT_SERVER_TYPE:-}" == "node" ]]; then
```

Comment:
if there are two RPs, how the nodes to assign to each RP ?

Reply:
This is called in function create-nodes-template(), which is run in rp for loop with different NODE_INSTANCE_PREFIX, RESOURCE_SERVER_KUBECONFIG,NODE_EXTRA_METADATA...